_data/master/navbars/networking.yml

```diff
@@ -4,10 +4,8 @@ order: 2
 toc:
 - title: Networking
   path: /networking/
-- title: Configuring IP-in-IP
-  path: /networking/ip-in-ip
-- title: Configuring VXLAN
-  path: /networking/vxlan
+- title: Connecting workloads that you do not control
```

Comment:
I think this title is wrong now - was meant to be "Connecting workloads across networks you do not control"

Reply:
I missed that somewhere in the rewrite. FIXED.